entries/jQuery.get.xml

```diff
@@ -2,6 +2,12 @@
 <entry type="method" name="jQuery.get" return="jqXHR">
   <title>jQuery.get()</title>
   <signature>
+    <added>2.2</added>
+    <argument name="settings" type="PlainObject" optional="false">
+      <desc>A set of key/value pairs that configure the Ajax request. All properties except for <code>url</code> are optional. A default can be set for any option with <a href="/jQuery.ajaxSetup/">$.ajaxSetup()</a>. See <a href="/jquery.ajax/#jQuery-ajax-settings">jQuery.ajax( settings )</a> for a complete list of all settings. Type will automatically be set to <code>POST</code></desc>
```

Comment:
I think we should be consistent within this paragraph, right now it's `$.ajaxSetup` and `jQuery.ajax`. I know we use both in docs but whichever is most common elsewhere should be used.